### PR TITLE
Handle default methods

### PR DESCRIPTION
**Pull request by juliusmik**

### CODE REVIEW

File: JNIBridge.java [hunk 1]
```diff
@@ -14,23 +15,73 @@ static Object newInterfaceProxy(final long ptr, final Class[] interfaces)
...
 	}
 
 	private static class InterfaceProxy implements InvocationHandler
 	{
 		private Object m_InvocationLock = new Object[0];
 		private long m_Ptr;
+		private Constructor<MethodHandles.Lookup> m_constructor;
 
-		public InterfaceProxy(final long ptr) { m_Ptr = ptr; }
+		@SuppressWarnings("unused")
```
Comment (Over17) [inline JNIBridge.java:28]: Is there anything unused in this method?

**Review by Over17 — APPROVED** [review]
LGTM

**Review by Over17 — APPROVED** [review]
LGTM

File: JNIBridge.java (hunk 1 above)
Comment (juliusmik) [reply JNIBridge.java:28]: "NoClassDefFoundError e" and "NoSuchMethodException e"

**Review by bitter — APPROVED** [review]
looks okay to me.

out of curiosity, did you try putting default method handling in native code?

**Pull request merged**